### PR TITLE
Fix a Bug adding or deleting a modification

### PR DESCRIPTION
**Pull request by sebastiansIT**
The new version 10.0.3 has a bug preventing me from adding and deleting modification from talent documents.

Accessing getData() is a async function but is used syncron.

### CODE REVIEW

File: src/item/item-sheet.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ export class ForbiddenLandsItemSheet extends ItemSheet {
 		super.activateListeners(html);
 		html.find(".add-modifier").click(async (ev) => {
 			ev.preventDefault();
-			let data = this.getData();
+			let data = await this.getData();
```
Comment (aMediocreDad) [inline src/item/item-sheet.js:83]: Thanks for finding these ones.

File: src/item/item-sheet.js [hunk 2]
```diff
@@ -94,7 +94,7 @@ export class ForbiddenLandsItemSheet extends ItemSheet {
 		});
 		html.find(".delete-modifier").click(async (ev) => {
 			ev.preventDefault();
-			let data = this.getData();
+			let data = await this.getData();
```
Comment (aMediocreDad) [inline src/item/item-sheet.js:97]: ^

File: src/legacy-styles/components/_editor-override.scss [hunk 3]
```diff
@@ -29,11 +29,10 @@
 	.editor,
```
Comment (aMediocreDad) [inline src/legacy-styles/components/_editor-override.scss:29]: Style changes here are somewhat unrelated to the PR, though warranted.

File: src/legacy-styles/components/_editor-override.scss [hunk 4]
```diff
@@ -29,11 +29,10 @@
...
 	.journal-entry-content {
 		font-family: var(--font-editor);
-		max-width: max(90%, 60ch);
-		margin-inline: auto;
+		margin: 1rem;
```
Comment (aMediocreDad) [inline src/legacy-styles/components/_editor-override.scss:32]: This here will create double scroll bars in Journal Entries. I suggest you remove it or change it to:

```suggestion
	article.journal-entry-page.text {
		margin-inline: 1rem;
	}
```

File: src/legacy-styles/components/_editor-override.scss [hunk 5]
```diff
@@ -64,6 +63,9 @@
 		p + p {
 			text-indent: 2rem;
 		}
+		p + p > * {
+			text-indent: 0;
+		}
```
Comment (aMediocreDad) [inline src/legacy-styles/components/_editor-override.scss:68]: I'm considering removing all text indents tbh. There have been several people suggesting removing them, and they only exist to imitate the source books.

**Review by aMediocreDad — APPROVED** [review]
Thanks for taking the initiative to open a PR 👍🏼 I have made some comments that I'd like to see resolved before merging.

File: src/legacy-styles/components/_editor-override.scss (hunk 3 above)
Comment (sebastiansIT) [reply src/legacy-styles/components/_editor-override.scss:29]: I'm so sorry. The CSS stuff is something I playing around with to fix other, more visual, stuff. It should not be part of this PR.

**Comment by sebastiansIT** [comment]
I have reverted the CSS stuff. Sorry. Now the PR should only contain the two changes in javascript.
